### PR TITLE
Feat/login page

### PR DESCRIPTION
**Pull request by angelique-m08**
create component LogPage and add css

### CODE REVIEW

File: conf.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const  mysql = require('mysql');
+const  connection = mysql.createConnection({
+  host :  'localhost',
+  user :  'angelique',
+  password :  'Bija2413!',
```
Comment (wasthishelpful) [inline conf.js:5]: thx for your password :)

this file should not be pushed... and should be part of a react project

**Comment by wasthishelpful** [comment]
conflicts to resolve :)

File: .gitignore [hunk 2]
```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+### config
+conf.js
```
Comment (wasthishelpful) [inline .gitignore:26]: useless since a react project doesn't have such a file :)

Comment (angelique-m08) [reply .gitignore:26]: I deleted conf file and deleted conf.js in .gitignore (error of repository :) )

Comment (wasthishelpful) [reply .gitignore:26]: they're still in the PR :)

**Comment by angelique-m08** [comment]
Requested changes done (I hope :) )

**Pull request merged**